hw/ip/clkmgr/data/clkmgr_testplan.hjson

```diff
@@ -153,7 +153,14 @@
       milestone: V2
       tests: []
     }
+    {
+      name: stress
+      desc: '''Create standard stress tests.'''
+      milestone: V3
+      tests: []
```

Comment:
```suggestion
      tests: [clkmgr_stress_all]
```

Reply:
I thought we not should add the tests until they are runnable.

Reply:
Agree with Matute here.

Reply:
I see. Thanks for correcting me.